client/src/index.js

```diff
@@ -3,4 +3,4 @@ import ReactDOM from 'react-dom';
 
 import App from './App';
 
-ReactDOM.render(<App />,document.getElementById("root"));
\ No newline at end of file
+ReactDOM.render(<App />,document.getElementById("root"));
```

Comment:
What was the problem here?

Reply:
Nothing, when I committed the changes VS Code added a new line at end of the file. That's why it's showing changes but actually, no change has been made